suite-native/transactions/src/components/TransactionDetail/TransactionDetailRow.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactNode } from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles/src';
+import { Box, Text } from '@suite-native/atoms/src';
+
+type TransactionDetailRowProps = {
+    title: string;
+    text?: string | number;
```

Comment:
Please avoid these optionals types for doing "OR". It basically saying that you can use just `title` without any other props which is clearly not indended. Solution to this is make two types:

```tsx
type TransactionDetailRowProps = {
  title: string;
  text: string | number;
} | {
  title: string;
  children: ReactNode
}
```

This will prevent anyone from misusage and types are exactly describing how component should be used. If you don't want to this manually is better to use type `RequireExactlyOne` from `type-fest` library which we use. https://github.com/sindresorhus/type-fest/blob/main/source/require-exactly-one.d.ts

Another option here could be to have just `children: ReactNode | string` and in render do `G.isString(children) ? <Text>{children}</Text> : {children}` (https://mobily.github.io/ts-belt/api/guards#isstring)

Reply:
fixed in [360bdb3](https://github.com/trezor/trezor-suite/pull/7370/commits/360bdb31f3189bc481292921ffd57db6f4609ee1)